scripts/quote-board.coffee

```diff
@@ -0,0 +1,42 @@
+# Description:
+#  Quote Board Interface
+#
+# Dependencies:
+#  None
+#
+# Commands:
+#  !quote <name> said <quote> - Adds that quote attributed to that person to the DB
+#  !quote_me_as <name> - Adds <name> to be your full name for quotes attributed to you
+#
+# Notes:
+#  None
+#
+# Author:
+#  @murph33
+
+QUOTE_URL = "https://quote-board.herokuapp.com/api/v1/quotes"
+PERSON_URL = "https://quote-board.herokuapp.com/api/v1/people"
+
+module.exports = (robot) ->
+  robot.hear /![q|Q]uote_me_as (.*)/i, (msg) ->
```

Comment:
Since your regexp is `//i` case insensitive, you wan't need to look for `[q|Q]`.

Do any of the other hubot commands use snake case keywords?